tools/crctab.c

```diff
@@ -0,0 +1,436 @@
+#include <stdint.h>
```

Comment:
There's no license declaration at the top of this file.

Also, this code doesn't match the code style of the rest of the repository.

Reply:
`crctab.c`, if you check the cksum tool `cksum.c`, it is generated in case you use -DCRCTAB:

```
/* Written by Q. Frank Xia, qx@math.columbia.edu.
   Cosmetic changes and reorganization by David MacKenzie, djm@gnu.ai.mit.edu.

  Usage: cksum [file...]

  The code segment between "#ifdef CRCTAB" and "#else" is the code
  which calculates the "crctab". It is included for those who want
  verify the correctness of the "crctab". To recreate the "crctab",
  do something like the following:

      cc -I../lib -DCRCTAB -o crctab cksum.c
      crctab > crctab.c

  This software is compatible with neither the System V nor the BSD
  'sum' program.  It is supposed to conform to POSIX, except perhaps
  for foreign language support.  Any inconsistency with the standard
  (other than foreign language support) is a bug.  */
```
The code is the same over all coreutils tools. Any ideas how to solve this issue?